controllers/ovndbcluster_controller.go

```diff
@@ -374,6 +375,15 @@ func (r *OVNDBClusterReconciler) reconcileNormal(ctx context.Context, instance *
 				err.Error()))
 			return ctrl.Result{}, err
 		}
+	} else if instance.Spec.DBType == ovnv1.SBDBType {
+		// This config map was created by the SB and it only needs to be deleted once
+		// since this reconcile loop can be done by the SB and the NB, filtering so only
+		// one deletes it.
+		Log.Info("NetworkAttachment is empty, deleting ExternalConfigMaps")
```

Comment:
`ExternalConfigMaps` what is that? It's not a class, not a variable. Should it use proper words then? "deleting external config map(s)"?

Reply:
Done